jcl/src/openj9.criu/share/classes/org/eclipse/openj9/criu/CRIUSupport.java

```diff
@@ -385,12 +385,6 @@ private void registerRestoreEnvVariables() {
 					String file = envFile.toAbsolutePath().toString();
 
 					try (BufferedReader envFileReader = new BufferedReader(new FileReader(file))) {
-						/* [IF JAVA_SPEC_VERSION < 17] */
-						SecurityManager manager = System.getSecurityManager();
-						if (manager != null) {
-							manager.checkRead(file);
-						}
-						/* [ENDIF] JAVA_SPEC_VERSION < 17 */
```

Comment:
Why remove the securitymanager check?  Isn't it still needed for Java 8 & 11?  (Though I'm on the fence about the value of providing this to 8...)

Reply:
The security check is done internally in `new FileReader(file)`

Reply:
> Why remove the securitymanager check? Isn't it still needed for Java 8 & 11? (Though I'm on the fence about the value of providing this to 8...)

See commit message: The FileReader already makes this check when reading from a file.

This is a redundant check.

Reply:
I will also comment that Java 17 still can have a functional security manager enabled (with some additional JVM configuration).  I'm not sure what `/* [IF JAVA_SPEC_VERSION < 17] */` does.  My guess is that it conditionally compiles this block of code in for Java < 17 targets.  I suggest you do not use this type of conditional blocks of code for Java 17 because it is still entirely possible to enable a security manager in Java 17.  It just isn't possible without an additional JVM option.

Reply:
We typically don't use deprecated APIs internally. JDK17 already outputs warning messages  if a security manager is used. Given that the the changes in JDK17 were made to wean users off of the security manager not sure if there is much benefit in allowing it here.